protocol/communities/community.go

```diff
@@ -1371,6 +1374,33 @@ func (o *Community) tokenPermissions() map[string]*CommunityTokenPermission {
 	return result
 }
 
+func (o *Community) PendingAndBannedMembers() map[string]CommunityMemberState {
+	result := make(map[string]CommunityMemberState)
+
+	// Non-privileged members should not see pending and banned members
+	if o.config.EventsData == nil || !o.IsPrivilegedMember(o.MemberIdentity()) {
+		return result
+	}
+
+	for _, bannedMemberID := range o.config.CommunityDescription.BanList {
+		result[bannedMemberID] = CommunityMemberBanned
+	}
+
+	for _, event := range o.config.EventsData.Events {
```

Comment:
To be consistent with how events are processed when obtaining the state of pending permissions, the first event should be counted rather than the last one. See: https://github.com/status-im/status-go/blob/6bcf5f1289f9160168574290cbd6f90dede3f8f6/protocol/communities/community.go#L1350

Reply:
fixed